### PR TITLE
Bumped smart_proxy_discovery to 1.0.2

### PR DESCRIPTION
**Pull request by lzap**

### CODE REVIEW

**Comment by domcleal** [comment]
The rest-client dep needs updating:

<pre>
  DIFFERENT runtime dependencies:
    1.0.1->1.0.2:
      * Updated:
            rest-client from: ["&gt;= 0"] to: ["&lt; 1.7", "&gt; 1.6.2"]
</pre>

(gem-compare if you haven't seen the README update)

**Comment by lzap** [comment]
Amended, nightly holds 1.6.7 which is fine.

File: rubygem-smart_proxy_discovery/rubygem-smart_proxy_discovery.spec [hunk 1]
```diff
@@ -15,7 +15,9 @@ Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
 Requires: ruby(rubygems)
 Requires: foreman-proxy >= 1.7.0
-Requires: rubygem(rest-client)
+Requires: rubygem(rest-client) >= 1.6.2
```
Comment (domcleal) [inline rubygem-smart_proxy_discovery/rubygem-smart_proxy_discovery.spec]: > 1.6.2

**Comment by lzap** [comment]
Yeah fixed.

**Comment by domcleal** [comment]
Thanks @lzap!  Building.

**Pull request merged**